### DISCUSSION TITLE
TYPO IN OPENING BANNER..

### DISCUSSION THREAD

**Discussion by RawbNasty** [community/community#29378] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/80489211/184516770-236deee1-d3df-4aeb-a2af-c81c99b6aed0.png)

**Comment by RawbNasty** [comment]
IN "EXPLORE GITHUB"

**Reply by radar-blip** [reply]
Where?

**Reply by ma7555** [reply]
looks good to me
